examples/lib/defaultHtmlStepUi.js

```diff
@@ -353,6 +351,12 @@ function DefaultHtmlStepUi(_sequencer, options) {
     else $step('.wasm-tooltip').fadeOut();
   }
 
+  function changeDimension(step){
```

Comment:
Can this function be called `updateDimensions`? The current name may be misleading. Please add JSDoc description comment for the function too. Thanks!